stub/stub.go

```diff
@@ -8,13 +8,20 @@ package stub
 
 import (
 	"bytes"
+	"flag"
 	"fmt"
 	"io"
+	"io/ioutil"
 	"log"
+	"os"
+	"path/filepath"
 	"runtime"
+	"runtime/pprof"
 	"sync/atomic"
 )
 
+var serverprofile = flag.String("serverprofile", "", "This is for specifying the prefix of the output file for the profile")
```

Comment:
I really dislike this paradigm defining flags that are not in or around the main function. I think it misses the point of having readable code, and the go community at large seems to agree. 

Could we move this inside Server struct?

> Only func main has the right to decide which flags are available to the user. 

[source](https://peter.bourgon.org/go-best-practices-2016/#top-tip-5)